src/scss/objects/_buttons.scss

```diff
@@ -48,8 +48,8 @@
 
     $btnGroup-outline-textColor         : $color-grey-50;
     $btnGroup-outline-border            : $color-grey-30;
-    $btnGroup-outline-active-border     : $color-secondary;
-    $btnGroup-outline-active-text-color : $color-secondary;
+    $btnGroup-outline-active-border     : $color-blue;
```

Comment:
I think this (and the line below) should use `$color-interactive-primary` (which maps to the same colour)

Reply:
I will change it in my next PR as I just merged this one :)